components/feature/readerview/src/main/java/mozilla/components/feature/readerview/view/ReaderViewControlsView.kt

```diff
@@ -45,6 +45,15 @@ interface ReaderViewControlsView {
      */
     fun asView(): View = (this as View)
 
+    /**
+     * Tries to inflate the view if needed.
+     *
+     * See: https://github.com/mozilla-mobile/android-components/issues/5491
+     *
+     * @return true if the inflation was completed, false if the view was already inflated.
+     */
+    fun tryInflate(): Boolean
```

Comment:
nit: should this be internal? I don't like that this is exposed to consumers.

Reply:
Can't make it internal. 🤷‍♂ 

This is where we have to give way for the implementation sake.